### PR TITLE
Convert ansyn to separated package 

### LINKED ISSUE #805: Convert ansyn to separated package
**I'm submitting a ...**  (check one with "x")

```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request
[ ] Task/UserStory
```

**Current behavior** 
ansyn app is not a spseparated package

**Expected behavior** 
ansyn app is a spseparated package

### PR DESCRIPTION
**Pull request by yairtawil**
resolve #805

### CODE REVIEW

File: src/app/packages/overlays/effects/overlays.effects.spec.ts [hunk 1]
```diff
@@ -16,7 +16,6 @@ import { OverlaysEffects } from './overlays.effects';
 import { OverlaysConfig, OverlaysService } from '../services/overlays.service';
 import { OverlayReducer, overlaysFeatureKey } from '../reducers/overlays.reducer';
 import { casesFeatureKey, CasesReducer } from '../../menu-items/cases/reducers/cases.reducer';
```
Comment (AmitMY) [inline src/app/packages/overlays/effects/overlays.effects.spec.ts]: @ ansyn

File: src/app/app/app-effects/effects/map.app.effects.ts [hunk 2]
```diff
@@ -617,7 +618,49 @@ export class MapAppEffects {
...
+		.map(toPayload)
+		.withLatestFrom(this.store$.select(casesStateSelector).pluck('selectedCase'), this.store$.select(mapStateSelector))
+		.map(([action, selectedCase, mapState]: [any, Case, IMapState]) => {
+			const updatedCase = {
+				...selectedCase,
+				state: {
+					...selectedCase.state,
+					maps: {
+						...selectedCase.state.maps,
+						data: [...mapState.mapsList],
```
Comment (yairtawil) [reply src/app/app/app-effects/effects/map.app.effects.ts]: this is so beautiful

File: src/app/app/ansyn/ansyn/ansyn.component.less [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import "~@ansyn/core/less/common";
+@import "../../../../../node_modules/@ansyn/core/less/common";
```
Comment (AmitMY) [inline src/app/app/ansyn/ansyn/ansyn.component.less]: oy vei zmir

File: src/app/app/app-effects/effects/map.app.effects.ts (hunk 2 above)
Comment (AmitMY) [inline src/app/app/app-effects/effects/map.app.effects.ts]: this is so ugly

**Pull request merged**